includes/blocks/class-sensei-lesson-blocks.php

```diff
@@ -12,13 +12,16 @@
 /**
  * Class Sensei_Lesson_Blocks
  */
-class Sensei_Lesson_Blocks {
+class Sensei_Lesson_Blocks extends Sensei_Blocks_Initializer {
 	/**
 	 * Sensei_Blocks constructor.
 	 */
 	public function __construct() {
+		parent::__construct( 'lesson' );
+
 		add_action( 'enqueue_block_assets', [ $this, 'enqueue_block_assets' ] );
 		add_action( 'enqueue_block_editor_assets', [ $this, 'enqueue_block_editor_assets' ] );
+		add_filter( 'sensei_use_sensei_template', [ $this, 'use_single_lesson_template' ] );
```

Comment:
I wonder if we could extract this part from this PR, so we can merge it earlier while we decide what we'll do about the template.

Reply:
We could that yes but I the thing is that I don't see any other possible way of handling this. As we did with the course page, whatever choice that we do we have to do it now (otherwise we will potentially need to maintain blocks based on two templates) and it wouldn't make much sense to follow a different approach for lessons. Maybe moving this code to another PR has no benefit and simply cost us time?

Reply:
I'll refactor to remove this for simply to allow merging of this PR.